bin/mdsctl

```diff
@@ -1235,7 +1175,7 @@ while getopts ":w:s:c:p:q-:h-:" opt; do
         sleep=*) configs[pause]="${OPTARG#*=}";;
         quiet) confgs[quiet]=true;;
         help) usage;;
-        *) [ "${OPTERR}" = 1 ] && [ "${optspec:0:1}" != ":" ] && echo "unknown option --${OPTARG}";;
+        *) [ "${OPTERR}" = 1 ] && [ "${optspec:0:1}" != ":" ] && error "unknown option --${OPTARG}";;
```

Comment:
What's the motivation for just echoing this versus throwing an error code?

Reply:
yeah bug ... good catch